test/assignment-tests.js

```diff
@@ -0,0 +1,80 @@
+import test from 'tape';
+import compose from '../examples/compose';
+
+const assignmentProps = [
+  'methods',
+  'properties',
+  'staticProperties',
+  'deepStaticProperties',
+  'propertyDescriptors',
+  'staticPropertyDescriptors'
+];
+
+const build = (num) => {
+  const composable = {};
+
+  assignmentProps.forEach(prop => {
+    composable[prop] = {
+      [num]: num,
+      override: num
+    };
+  });
+
+  return { compose: composable };
```

Comment:
It would be a better idea to make `compose` a function like it is defined in the spec.

```js
Object.assign(() => {}, composable);
```